### DISCUSSION TITLE
New server, API (bun version) and personal

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#61] [Announcements]
Hello everyone, We recently moved our API to the actual VPS now thanks to **adgstudios.co.za** for gifting me a server. You might find the index url returning 404 because we haven't configured it correctly yet so please be patient. 

We also released the Bun version of our API completely rewriting the code in Typescript and switching from expressjs to elysiajs [NEW API](https://github.com/amvstrm/api-bun) (The code is still in beta and missing some routes and functions).

**NYT92's update** : For those who haven't read my profile, I am going to be taking a national exam in October so next month I might quit updating the new code completely but I only do critical updates to the code if it breaks the website & API and still actively helping the issue and answering the question.

Thank you for your time reading this.

**Comment by Josumaru** [comment]
I'm having trouble deploying to vercel, do you have any documentation on that or is vercel.josn required? thanks
